core/services/ocr2/plugins/promwrapper/factory.go

```diff
@@ -0,0 +1,41 @@
+package promwrapper
+
+import (
+	"math/big"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+)
+
+var _ types.ReportingPluginFactory = &promFactory{}
+
+type ChainType string
+
+const (
+	EVM ChainType = "evm"
+)
```

Comment:
I think we already have chain type constants somewhere:

https://github.com/smartcontractkit/chainlink/blob/84ac9637da0f9ab339b9e71ca3a8e53a8e6f5d8e/core/services/relay/relay.go#L3

Reply:
thanks for the suggestion. yeah I initially thought of defining own types for promwrapper but probably better to avoid duplication